sdk/search/azure-search-documents/tests/test_search_index_client.py

```diff
@@ -88,24 +113,32 @@ def test_get_document_count(self, mock_count):
         client.get_document_count()
         assert mock_count.called
         assert mock_count.call_args[0] == ()
-        assert mock_count.call_args[1] == {}
+        assert len(mock_count.call_args[1]) == 1
+        assert mock_count.call_args[1]["headers"] == client._headers
+
 
     @mock.patch(
         "azure.search.documents._index._generated.operations._documents_operations.DocumentsOperations.get"
     )
-    def test_get_document_count(self, mock_get):
+    def test_get_document(self, mock_get):
         client = SearchIndexClient("endpoint", "index name", CREDENTIAL)
         client.get_document("some_key")
         assert mock_get.called
         assert mock_get.call_args[0] == ()
-        assert mock_get.call_args[1] == {"key": "some_key", "selected_fields": None}
+        assert len(mock_get.call_args[1]) == 3
+        assert mock_get.call_args[1]["headers"] == client._headers
+        assert mock_get.call_args[1]["key"] == "some_key"
+        assert mock_get.call_args[1]["selected_fields"] == None
```

Comment:
This was preventing the previous test from actually getting run (copy-pasta, overwrote the func name)